### PR TITLE
fix(cli): signing type option issue

### PR DESCRIPTION
**Pull request by liuwin7**
There are two revision.
The first one is to correct the `commander` use case, if a option is named with b `-`, it will change to the Camel Case.
The second is to load  the all android build options, not only the options for `apksigner `.

### CODE REVIEW

File: ios/Capacitor/Capacitor/JS.swift [hunk 1]
```diff
@@ -19,13 +19,6 @@ internal struct JSCall {
     let pluginId: String
     let method: String
     let callbackId: String
-
-    init(options: [String: Any], pluginId: String, method: String, callbackId: String) {
```
Comment (markemer) [inline ios/Capacitor/Capacitor/JS.swift:23]: What's this in here for?

File: cli/src/index.ts [hunk 2]
```diff
@@ -157,7 +157,7 @@ export function runProgram(config: Config): void {
     )
     .addOption(
       new Option(
-        '--signing-type <signingtype>',
+        '--signingtype <signingtype>',
```
Comment (markemer) [inline cli/src/index.ts:160]: This is a breaking change - can we instead keep it as is and fix the variables?

File: core/native-bridge.ts [hunk 3]
```diff
@@ -1038,6 +1038,7 @@ const initBridge = (w: any): void => {
       });
     };
 
+    // eslint-disable-next-line @typescript-eslint/no-unused-vars
```
Comment (markemer) [inline core/native-bridge.ts:1041]: I'm all for this, but we might want to put this in another PR.

**Review by markemer — CHANGES_REQUESTED** [review]
Just a few questions, but I think we can make this change.

File: ios/Capacitor/Capacitor/JS.swift (hunk 1 above)
Comment (liuwin7) [reply ios/Capacitor/Capacitor/JS.swift:23]: Because I run `npm run fmt`.
For `Swift` the struct's `init` function is default created when compiled, no need to create explicitly.

File: core/native-bridge.ts (hunk 3 above)
Comment (liuwin7) [reply core/native-bridge.ts:1041]: Ok, I did this mainly to pass the review reboot.

File: cli/src/index.ts (hunk 2 above)
Comment (liuwin7) [reply cli/src/index.ts:160]: It's ok.
But, it is surely a breaking change same as using the `--signing-type` instead of `--signingtype` following the convention initially.

File: core/native-bridge.ts (hunk 3 above)
Comment (markemer) [reply core/native-bridge.ts:1041]: Did npm run fmt do this? If so, lets keep it.

File: ios/Capacitor/Capacitor/JS.swift (hunk 1 above)
Comment (markemer) [reply ios/Capacitor/Capacitor/JS.swift:23]: Yeah - I'm not sure why they're in there - `npm run fmt` catching it probably makes this ok to include.

File: core/native-bridge.ts (hunk 3 above)
Comment (liuwin7) [reply core/native-bridge.ts:1041]: No for `npm run fmt`, but `npm run lint` need.

**Review by markemer — APPROVED** [review]

**Review by markemer — APPROVED** [review]

**Review by giralte-ionic — APPROVED** [review]

**Comment by liuwin7** [comment]
Why still not merged yet after 2 versions have been released?

**Review by IT-MikeS — APPROVED** [review]
